lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_retrospectives.py

```diff
@@ -202,6 +202,8 @@ def create_encounter(db: DatabaseSession,
     encounter_class = create_encounter_class(record)
     encounter_status = create_encounter_status(record)
 
+    encounter_reason_codes = parse_icd10_codes(record, ('icd10_primary', 'icd10_secondary'))
```

Comment:
ICD10 codes is an interesting one to bring in. In the FHIR docs, I'm seeing them used for various things related to billing/claims. Did you find prior art for bringing them in as encounter reason code? On the encounter, it might also be a good fit for diagnosis, but I'm not sure what is the most standard way to approach incorporating these into the FHIR bundle. http://www.hl7.org/FHIR/encounter-definitions.html#Encounter.reasonCode

Reply:
No, I didn't have any previous reference for this, and I wasn't quite sure where else to put them. Most of the ones I looked up were symptoms, so I though encounterCode would be a logic place. Now that I'm looking at FHIR docs, it looks like `ChargeItem.reason` may be a better place to put them: https://build.fhir.org/valueset-icd-10.html (build version of FHIR has a more complete list of codes, but current version does not: http://hl7.org/fhir/valueset-icd-10.html)

Reply:
After further review, it looks like `Condition.code` is another (and maybe the best) option. I found a couple of references outside of FHIR documentation where it's being used that way, and that location would make more sense than `ChargeItem.reason`:

https://fhir.cerner.com/millennium/dstu2/general-clinical/condition/
https://mednet.swiss/fhir/StructureDefinition-mni-familyMemberHistory.html

Reply:
`Condition.code` looks great - good find.